dev-docs/bidders/appnexus.md

```diff
@@ -56,6 +56,7 @@ All AppNexus placements included in a single call to `requestBids` must belong t
 | `pubClick`          | optional | Specifies a publisher-supplied URL for third-party click tracking. This is just a placeholder into which the publisher can insert their own click tracker. This parameter should be used for an unencoded tracker. This parameter is expected to be the last parameter in the URL. Please note that the click tracker placed in this parameter will only fire if the creative winning the auction is using AppNexus click tracking properly.                                  | `'http://click.adserver.com/'`                        | `string`         |
 | `extInvCode`        | optional | Specifies predefined value passed on the query string that can be used in reporting. The value must be entered into the system before it is logged.                           | `'10039'`                                             | `string`         |
 | `externalImpId`     | optional | Specifies the unique identifier of an externally generated auction.                                                                                                           | `'bacbab02626452b097f6030b3c89ac05'`                  | `string`         |
+| `generate_ad_pod_id`| optional | Signal to AppNexus to split impressions by ad pod and add unique ad pod id to each request. Specific to long form video endpoint only.                                        | `true`                                                | `boolean`        |
```

Comment:
May want to change it to `Intended for long form video endpoint only.`. There doesn't seem to be anything to prevent it being used in non-long form video, but also does not make sense in this context either. Perhaps someone will come up with a smart twist that uses this on the main openrtb endpoint.

Reply:
It will be ignored for non-long form video because the code branch is only activated for /video traffic. I like the idea of adding the long form video endpoint only note. I'd also think it's important to note this is only supported by PBS and not PBJS (there's no need on the PBJS side as far as I'm aware).

Reply:
Modified.